doc/api/process.md

```diff
@@ -1593,6 +1593,26 @@ Will generate:
 When using [`Worker`][] threads, `rss` will be a value that is valid for the
 entire process, while the other fields will only refer to the current thread.
 
+The `process.memoryUsage()` method iterate over each page to gather
+informations about memory usage which can be slow depending on the
+program memory allocations.
+
+## `process.memoryUsage.rss()`
+
+* Returns: {integer}
+
+The `process.memoryUsage.rss()` method returns an integer representing the
+Resident Set Size (RSS).
+
+The Resident Set Size, is the amount of space occupied in the main
+memory device (that is a subset of the total allocated memory) for the
+process, including all C++ and JavaScript objects and code.
```

Comment:
Maybe mention that this is the same as the `rss` property returned by `memoryUsage()`.